lab-nicholasc/README.md

```diff
@@ -0,0 +1,11 @@
+#index.js
```

Comment:
-- `node index.js` starts the server -- 

would be helpful documentation for a newbie or someone who doesn't necessarily know the terminal commands to start the server